modules/claims_api/app/sidekiq/claims_api/poa_updater.rb

```diff
@@ -30,7 +30,9 @@ def perform(power_of_attorney_id, rep = nil) # rubocop:disable Metrics/MethodLen
 
         ClaimsApi::VANotifyJob.perform_async(poa_form.id, rep) if vanotify?(poa_form.auth_headers, rep)
 
-        ClaimsApi::PoaVBMSUpdater.perform_async(poa_form.id) if enable_vbms_access?(poa_form:)
+        if enable_vbms_access?(poa_form:) && update_poa_access?(poa_form.auth_headers)
```

Comment:
Iʼm not sure the extra helper methods add any value. Could we instead just say:

```suggestion
        if enable_vbms_access?(poa_form:) && auth_headers['dependent'].blank?
```

?

Reply:
@tycol7  Yep makes sense,  b26e5c3